packages/synapse-react-client/src/components/ImageCardGridWithLinks/ImageCardGridWithLinks.tsx

```diff
@@ -0,0 +1,216 @@
+  sql: string
+  title: string
+  summaryText: string
+}
+
+type ImageCardProps = {
+  card: Row
+  index: number
+}
+
+enum ExpectedColumns {
+  LINKTEXT = 'LinkText',
+  LINK = 'Link',
+  IMAGE = 'Image',
+}
+
+function ImageCardGridWithLinks(props: ImageCardGridWithLinksProps) {
+  const { sql, title, summaryText } = props
+  const { accessToken } = useSynapseContext()
+  const [images, setImages] = useState<string[] | undefined>()
+
+  const entityId = parseEntityIdFromSqlStatement(sql)
+  const queryBundleRequest: QueryBundleRequest = {
+    partMask:
+      SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
+      SynapseConstants.BUNDLE_MASK_QUERY_RESULTS,
+    concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
+    entityId,
+    query: {
+      sql,
+    },
+  }
+
+  const { data: queryResultBundle, isLoading } =
+    useGetQueryResultBundle(queryBundleRequest)
+
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        const imageColumnIndex = getFieldIndex(
+          ExpectedColumns.IMAGE,
+          queryResultBundle,
+        )
+        const image = (queryResultBundle?.queryResult!.queryResults.rows.map(
+          el => el.values[imageColumnIndex],
+        ) ?? []) as string[]
+        const imageFileHandleIds = image.filter(
+          v => v != null && v !== undefined,
+        )
+        if (imageFileHandleIds.length === 0) {
+          return
+        }
+        const fileHandleAssociationList: FileHandleAssociation[] =
+          imageFileHandleIds.map(fileId => {
+            return {
+              associateObjectId: entityId,
+              associateObjectType: FileHandleAssociateType.TableEntity,
+              fileHandleId: fileId,
+            }
+          })
+        const batchFileRequest: BatchFileRequest = {
+          includeFileHandles: false,
+          includePreSignedURLs: true,
+          includePreviewPreSignedURLs: false,
+          requestedFiles: fileHandleAssociationList,
+        }
+        const files = await getFiles(batchFileRequest, accessToken)
+        setImages(
+          files.requestedFiles
+            .filter(el => el.preSignedURL !== undefined)
+            .map(el => el.preSignedURL!),
+        )
+      } catch (e) {
+        console.error('Error on get data', e)
+      }
+    }
+    getData()
+  }, [entityId, accessToken, queryResultBundle])
+
+  const dataRows = queryResultBundle?.queryResult!.queryResults.rows ?? []
+  const linkColumnIndex = getFieldIndex(ExpectedColumns.LINK, queryResultBundle)
+  const linkTextColumnIndex = getFieldIndex(
+    ExpectedColumns.LINKTEXT,
+    queryResultBundle,
+  )
+
+  const ImageCard = ({ card, index }: ImageCardProps) => (
+    <Grid
+      item
+      xs={12}
+      sm={6}
+      md={4}
+      key={card.rowId}
+      sx={{ height: '245px', paddingTop: '24px', paddingLeft: '24px' }}
+    >
+      {isLoading ? (
+        <Skeleton variant="rectangular" height={221} width="100%" />
+      ) : (
+        <>
```

Comment:
unlike the publications component, this one does seem like it might be appropriate to use the `Card` component as a wrapper, with prop `raised: false`. You may consider using that and seeing if you can end up removing unnecessary styles. I would keep it if it ends up making this code more simple.